frontend/src/components/elements/ImageList/ImageList.tsx

```diff
@@ -26,6 +26,8 @@ import "./ImageList.scss"
 
 export interface Props {
   width: number
+  height?: number
```

Comment:
I would prefer keeping the props sorted, though we don't have a rule for that. Let me know what you think.

Reply:
Sorted in which way? but yes I'm agree

Reply:
alphabetically?

Reply:
Should I add a task for that?

Reply:
Dissenting opinion: I don't think "alphabetical sorting" is a good policy for props, member variables, and what-have-you. There's often a more meaningful semantic sorting that the author imparts.